basic/src/test/java/org/jboss/hal/testsuite/test/runtime/server/configurationchanges/ConfigurationChangesTestCase.java

```diff
@@ -0,0 +1,191 @@
+    private static Address serviceAddress;
+    private static ConfigurationChangesResource changesResource;
+
+    @BeforeClass
+    public static void beforeClass() {
+        client = ManagementClientProvider.createOnlineManagementClient();
+        operations = new Operations(client);
+        serviceAddress = client.options().isDomain ?
+                Address.host(client.options().defaultHost)
+                        .and("core-service", "management")
+                        .and("service", "configuration-changes") :
+                Address.coreService("management")
+                        .and("service", "configuration-changes");
+        changesResource = new ConfigurationChangesResource(client);
+    }
+
+    @Before
+    public void before() throws IOException, OperationException {
+        //make sure the service is up and running
+        if (!operations.exists(serviceAddress)) {
+            operations.add(serviceAddress, Values.of("max-history", 100)).assertSuccess();
+        }
+        page.navigate();
+    }
+
+    @AfterClass
+    public static void afterClass() throws IOException, OperationException, TimeoutException, InterruptedException {
+        try {
+            operations.removeIfExists(serviceAddress);
+            Administration administration = new Administration(client);
+            administration.restartIfRequired();
+            administration.reloadIfRequired();
+        } finally {
+            client.close();
+        }
+    }
+
+    @Test
+    public void disableAndEnableTracking() throws Exception {
+        page.disable();
+        new ResourceVerifier(serviceAddress, client)
+                .verifyDoesNotExist();
+
+        page.openEnableConfigurationChangesDialog()
+                .maxHistory(100)
+                .clickSave();
+        new ResourceVerifier(serviceAddress, client)
+                .verifyAttribute("max-history", 100);
+    }
+
+    @Test
+    //TODO remove https://issues.jboss.org/browse/HAL-1182 message
+    public void allRelevantRecordsAreShown() throws IOException, ParseException {
+        Assert.assertTrue("Some unexpected differences found! Either some changes are not displayed or some extra appeared. Some could fail due to the https://issues.jboss.org/browse/HAL-1182",
+                isChangesListsEqual(changesResource.getAllConfigurationChanges(),
+                                    page.getConfigurationChangesTable().getAllConfigurationChanges()));
+    }
+
+    @Test
+    //TODO remove https://issues.jboss.org/browse/HAL-1182 message
+    public void filterConfigurationChangesCapabilities() throws Exception {
+        List<ConfigurationChange> filtered_S_configurationChanges = page.filter().setFilter("s").getAllConfigurationChanges();
+        Assert.assertTrue("Some unexpected differences found! Either some changes are not displayed or some extra appeared. Some could fail due to the https://issues.jboss.org/browse/HAL-1182",
```

Comment:
You could update the sentence to describe the situation in which case it fails due HAL-1182. Also putting the message to some constant might be good idea as you are repeating the same assert message multiple times.